huaweicloud/resource_huaweicloud_cce_node_attach.go

```diff
@@ -0,0 +1,382 @@
+package huaweicloud
+
+import (
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/huaweicloud/golangsdk/openstack/cce/v3/nodes"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/logp"
+)
+
+func ResourceCCENodeAttachV3() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceCCENodeAttachV3Create,
+		Read:   resourceCCENodeV3Read,
+		Update: resourceCCENodeV3Update,
+		Delete: resourceCCENodeAttachV3Delete,
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(20 * time.Minute),
+			Delete: schema.DefaultTimeout(20 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"cluster_id": {
```

Comment:
mising **region**

Reply:
This resource is a relation type resource. It won't create any instance. So region will not work.

Reply:
fixed